src/ArrayToXml.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Spatie\ArrayToXml;
 
+use \Closure;
```

Comment:
Could you import this?

Reply:
With import you mean don't use `use \Closure` but use `if ($value instanceof \Closure) {`?

Reply:
Sorry, that was indeed vague. Just remove the starting `\`

Reply:
@freekmurze that should be fixed in the latest commit